dubbo-common/src/test/java/org/apache/dubbo/common/utils/LFUCacheTest.java

```diff
@@ -73,36 +73,6 @@ public void testDefaultCapacity() throws Exception {
         assertThat(cache.getCapacity(), equalTo(1000));
     }
 
-    @Test
```

Comment:
why remove these test codes？

Reply:
I just  revert #7967，The removed code is useless